ainow/templates/ainow/sponsorship.html

```diff
@@ -0,0 +1,412 @@
+          </div>
+          <h3 class="sponsorship-tier__heading">Drinks reception sponsor</h3>
+          <h4 class="sponsorship-tier__amount">$4,000</h4>
+          <ul class="breathier-list sponsorship-tier__perks">
+            <li>Your branding / logo / company url on all conference materials, conference website and promotional media</li>
+            <li>Logo displayed during the drinks reception</li>
+            <li>Your promotional materials displayed on tables during drinks reception (provided by sponsor)</li>
+            <li>Social media announcement when sponsorship is confirmed</li>
+            <li>One (1) free TICTeC ticket for staff</li>
+          </ul>
+        </div>
+      </div>
+      <div class="col-md-6">
+        <div class="sponsorship-tier--minor">
+          <div class="sponsorship-tier__badge">
+            <svg xmlns="http://www.w3.org/2000/svg" width="43" height="50" viewBox="0 0 43 50"><path fill="#9FDB50" fill-rule="evenodd" d="M21.5 0L43 12.5v25L21.5 50 0 37.5v-25L21.5 0zm4.62 32.747h2.296V19.06H26.12v13.688zm-1.34-14.831h-5.824c-.027-.294.004-.598.156-.85.345-.576 1.287-.874 2.74-.865 1.464.01 2.421.327 2.77.915.145.248.178.531.159.8zm-9.466 14.831h2.296V19.06h-2.296v13.688zM29.85 17.916h-3.87c.02-.451-.05-.942-.32-1.403-.58-.99-1.857-1.499-3.799-1.513-1.935-.018-3.2.477-3.781 1.456-.285.475-.353.99-.325 1.46H13.88a2.883 2.883 0 0 0-2.88 2.88V31.12A2.883 2.883 0 0 0 13.88 34h15.97a2.883 2.883 0 0 0 2.88-2.88V20.796a2.883 2.883 0 0 0-2.88-2.88z"/></svg>
+          </div>
+          <h3 class="sponsorship-tier__heading">Travel grantee sponsor</h3>
+          <h4 class="sponsorship-tier__amount">$2,000</h4>
+          <ul class="breathier-list sponsorship-tier__perks">
+            <li>Your branding / logo / company url on all conference materials, conference website and promotional media</li>
+            <li>Social media announcement when sponsorship is confirmed</li>
+          </ul>
+        </div>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <p class="sponsorship-benefits-callout">If you have ideas for additional sponsorship benefits that are not listed, we would be happy to discuss how we can accommodate those. We are also happy to discuss the creation of bespoke sponsorship packages. <a href="#">Just get in touch</a>.</p>
```

Comment:
This should be a `mailto:` link to @Gemmamysoc

Reply:
yep - she gave my an address for it, adding now